lms/envs/common.py

```diff
@@ -2174,6 +2174,13 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
```

Comment:
Essentially, the question is "will other APIs pick these settings up by default once they are defined/available?"